macros/get_lead_columns.sql

```diff
@@ -7,9 +7,38 @@
     {"name": "first_name", "datatype": dbt.type_string()},
     {"name": "id", "datatype": dbt.type_int(), "alias": "lead_id"},
     {"name": "last_name", "datatype": dbt.type_string()},
-    {"name": "updated_at", "datatype": dbt.type_timestamp(), "alias": "updated_timestamp"}
+    {"name": "updated_at", "datatype": dbt.type_timestamp(), "alias": "updated_timestamp"},
+    {"name": "phone", "datatype": dbt.type_string()},
+    {"name": "main_phone", "datatype": dbt.type_string()},
+    {"name": "mobile_phone", "datatype": dbt.type_string()},
+    {"name": "company", "datatype": dbt.type_string()},
+    {"name": "inferred_company", "datatype": dbt.type_string()},
+    {"name": "address_lead", "datatype": dbt.type_string()},
+    {"name": "address", "datatype": dbt.type_string()},
+    {"name": "city", "datatype": dbt.type_string()},
+    {"name": "state", "datatype": dbt.type_string()},
+    {"name": "state_code", "datatype": dbt.type_string()},
+    {"name": "country", "datatype": dbt.type_string()},
+    {"name": "country_code", "datatype": dbt.type_string()},
+    {"name": "postal_code", "datatype": dbt.type_string()},
+    {"name": "billing_street", "datatype": dbt.type_string()},
+    {"name": "billing_city", "datatype": dbt.type_string()},
+    {"name": "billing_state", "datatype": dbt.type_string()},
+    {"name": "billing_state_code", "datatype": dbt.type_string()},
+    {"name": "billing_country", "datatype": dbt.type_string()},
+    {"name": "billing_country_code", "datatype": dbt.type_string()},
+    {"name": "billing_postal_code", "datatype": dbt.type_string()},
+    {"name": "inferred_city", "datatype": dbt.type_string()},
+    {"name": "inferred_state_region", "datatype": dbt.type_string()},
+    {"name": "inferred_country", "datatype": dbt.type_string()},
+    {"name": "inferred_postal_code", "datatype": dbt.type_string()},
+    {"name": "inferred_phone_area_code", "datatype": dbt.type_string()},
+    {"name": "anonymous_ip", "datatype": dbt.type_string()},
+    {"name": "unsubscribed", "datatype": dbt.type_boolean(), "alias": "is_unsubscribed"},
+    {"name": "email_invalid", "datatype": dbt.type_boolean(), "alias": "is_email_invalid"},
```

Comment:
Could these alias's technically be breaking? Is there a scenario where `unsubscribed` and/or `email_invalid` exist in the staging and other models at this moment, but then after this update those fields are now gone and replaced with the `is_*` rename? See the below screenshot where I run into this scenario.

![image](https://github.com/fivetran/dbt_marketo_source/assets/74217849/d4f7023c-fe18-4a87-8d88-95d93956fc9f)

Reply:
If this is the case then unfortunately we will want to make this a breaking change and document it in the CHANGELOG and also persist this downstream in a transform release.

Reply:
ah yeah you're totally right, will adjust

Reply:
i realize now that we needed a transform PR anyways, in order to add these columns' yml descriptions